internal/typename/typetest.go

```diff
@@ -0,0 +1,49 @@
+package typename
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+)
+
+// Declare declares a unit test-suite for functions "Of()" defined in "goreflect"
+// and "gotypes" packages.
+func Declare(dir string, fn func(string) string) {
```

Comment:
```suggestion
func Declare(fn func(string) string) {
```

Can probably just hardcode `dir`, I would think?

Reply:
That's what I had in my first version of `Declare()`, I will hardcode it again 👍

Reply:
Should be fixed in https://github.com/dogmatiq/configkit/pull/115/commits/0bebaa4c6179a3db77be5a9e858cca5a519190f7